### DISCUSSION TITLE
How to install github files after downloading

### DISCUSSION THREAD

**Discussion by UNDELA** [community/community#23043] [New to GitHub]
<p>I have downloaded github’s gnu mcu eclipse of risc5…I m not understanding how to install …which file to install the application and the promt window for executing commands. Please do the needful in installing and executing the same by providing detailed procedure</p>

**Comment by ThirdEyeRose** [comment, marked as answer]
<p>Hi <a class="mention" href="/undela">@undela</a>!</p>
<p>I moved your post into the <strong>How to use Git and GitHub</strong> board since your general question was about installing files that you downloaded from GitHub. </p>
<p>The short answer is that there is no standarrdized method in which to install files downloaded from GitHub. Files hosted on GitHub are created by individuals and organizations around the world and all have unique instructions for use based upon what sort of code is being hosted and who created it.</p>
<p>Without knowing the repository that you downloaded these files from, I can’t point you to specific instructions in that repository which are usually found in the README file, but I did find this article on <a href="https://gnu-mcu-eclipse.github.io/toolchain/riscv/install/" rel="nofollow noopener">How to install the RISC-V toolchain</a> which might be helpful.</p>
<p>Hope that helps!</p>
